open_humans/urls.py

```diff
@@ -137,6 +141,11 @@
     path('account/login/oauth2/', views.OAuth2LoginView.as_view(),
          name='account-login-oauth2'),
 
+    # use our own template.
+    path('account/social/signup/',
+         account_views.SocialSignupView.as_view(),
+         name='SocialSignupView'),
```

Comment:
There's inconsistency here in `name` declarations and what you're doing elsewhere.

The form for this page is POSTing to the URL name `socialaccount_signup`. This is defined by django-allauth to be this path... and that ends up working, as you've overridden the path django-allauth usually uses (but it will still happily point you to it).

But this `name` isn't being used at all, and is inconsistent with the naming scheme we've used...